<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>555</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#FF7F50</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>331</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ваши напоминания</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addEventButt">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>259</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> display: inline-block;
    text-decoration: none;
    background-color: #ffa852;
    color: #006089;
    border: 3px solid #d9812f;
    border-radius: 5px;
    font-size: 16px;
    padding: 7px 10px; 
    transition: all 0.8s ease;</string>
   </property>
   <property name="text">
    <string>Добавить событие</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>350</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>53</number>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>Строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Иван</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Напоминание</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="clearButt">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>490</y>
     <width>259</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> display: inline-block;
    text-decoration: none;
    background-color: #ffa852;
    color: #006089;
    border: 3px solid #d9812f;
    border-radius: 5px;
    font-size: 16px;
    padding: 7px 10px; 
    transition: all 0.8s ease;</string>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="checkLogButt">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"> display: inline-block;
    text-decoration: none;
    background-color: #ffa852;
    color: #006089;
    border: 3px solid #d9812f;
    border-radius: 5px;
    font-size: 16px;
    padding: 7px 10px; 
    transition: all 0.8s ease;</string>
      </property>
      <property name="text">
       <string>Посмтотреть входы</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addRememberButt">
      <property name="styleSheet">
       <string notr="true"> display: inline-block;
    text-decoration: none;
    background-color: #ffa852;
    color: #006089;
    border: 3px solid #d9812f;
    border-radius: 5px;
    font-size: 16px;
    padding: 7px 10px; 
    transition: all 0.8s ease;</string>
      </property>
      <property name="text">
       <string>Добавить участников</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ExitButt">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"> display: inline-block;
    text-decoration: none;
    background-color: #ffa852;
    color: #006089;
    border: 3px solid #d9812f;
    border-radius: 5px;
    font-size: 16px;
    padding: 7px 10px; 
    transition: all 0.8s ease;</string>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
